app/static/index.css

```diff
@@ -168,10 +168,6 @@ li a:hover {
   font-weight: 300;
 }
 
-.search-btn {
-  width: 10.5%;
-}
-
```

Comment:
@azaria-dedmon please make this change below
`.search {
  margin-top: 1.1%;
  margin-left: 0.5%;
  margin-bottom: 2%;
  width: 300px;
  height: 27px
}

.search-btn {
  margin-top: 1.1%;
  margin-left: 0.5%;
  height: 27px;
}`